src/systems/sensors/Sensors.cc

```diff
@@ -362,6 +411,19 @@ Sensors::~Sensors()
   this->dataPtr->Stop();
 }
 
+void Sensors::SetSameProcess(bool _sameProcess)
```

Comment:
one idea is to add a `EntityComponentManager::SameProcess()` function so we could access this property from all class. Maybe also a bit hacky though.

Reply:
This is a new method of the class `ISystemConfigure`. All system plugins that override this method should get access to this data.

https://github.com/ignitionrobotics/ign-gazebo/blob/5521b407e41376643d2064a79643f52673c73f4b/include/ignition/gazebo/System.hh#L102-L105